rrfs-test/testinput/rrfs_fv3jedi_bumploc_2022052619.yaml

```diff
@@ -0,0 +1,86 @@
+geometry:
+ fms initialization:
+   namelist filename: DataFix/fmsmpp.nml
+   field table filename: DataFix/field_table
+ namelist filename: DataFix/input_lam_C775.nml
+ akbk: DataFix/fix/akbk61.nc
+ field metadata override: Data/fieldmetadata/tlei-gfs-restart.yaml
+ # input.nml
+ npz: 60
+ layout: [8,10]
+ io_layout: [1,1]
+ ntiles: 1 
+background:
+  datetime: 2022-05-26T19:00:00Z
+  filetype: fms restart
+  datapath: Data/bkg
+  filename_core: fv3_dynvars.nc
+  filename_trcr: fv3_tracer.nc
+  filename_sfcd: fv3_sfcdata.nc
+  filename_cplr: coupler.res
+  state variables: [air_temperature,surface_pressure] 
+background error:
+  covariance model: SABER
+  saber central block:
+    saber block name: BUMP_NICAS
+    calibration:
+      general:
+        universe length-scale: 250.0e3
+      io:
+        files prefix: Data/bump_new/fv3jedi_bumpparameters_nicas_lam_atm
+        alias:
+        - in code: air_temperature
+          in file: fixed_250km_0.3
+        - in code: surface_pressure
+          in file: fixed_250km
+
+      drivers:
+        multivariate strategy: univariate 
+        compute nicas: true
+        write local nicas: true
+      nicas:
+        resolution: 6
+        explicit length-scales: true
+        horizontal length-scale:
+        - groups:
+          - air_temperature
+          value: 250000.0
+        - groups:
+          - surface_pressure
+          value: 250000.0
+
+        vertical length-scale:
+        - groups:
+          - air_temperature
+          value: 0.3
```

Comment:
Line 55, @TingLei-NOAA Do you know how to set this value if we want to set the vertical length-scale to be 3 vertical levels?

Reply:
@guoqing-noaa  I don't know if bump allows other specification method, like in the grid unit,  for the vertical scale. Maybe, you could ask the question in saber.

Reply:
For MPAS-JEDI, the units for vertical localization can be set in the geometry section of the yaml you use for BUMP. However, I am not seeing this option in any of the code for FV3-JEDI, so I am not sure if the same functionality is available there. 

Example for MPAS-JEDI: 

```
geometry:
  nml_file: "./namelist.atmosphere_15km"
  streams_file: "./streams.atmosphere_15km"
  deallocate non-da fields: true
  bump vunit: "height" # vertical unit for vertical localization [modellevel, height]
```

Reply:
The bump yaml for MPAS-JEDI is currently included in the staged directory with the ensemble localization files. But it's a good idea to include it in the repository.

Reply:
I just added a bumploc yaml for mpas-jedi to this PR. Hope that worked - I haven't added a commit to another user's PR before.

Reply:
@SamuelDegelia-NOAA  Thanks for adding this mpas-jedi bump yaml.  
@guoqing-noaa   it would like to highlight that a rather large horizontal scale 1000km is used in this yaml (and in this mpas-jedi test case in rdasapp) because a realistic length scale like 200km (with 36 mpi ranks used by this test case)  would make the bump tool fail to finish in 8 hours clock time limit.     Yongzuo is testing a new bump function ( https://github.com/JCSDA-internal/saber/pull/833.)  and hopefully to resolve this issue.